app/templates/application.hbs

```diff
@@ -6,7 +6,13 @@
   {{!-- <%= javascript_include_tag 'common-modern' %> --}}
 <!--<![endif]-->
 
-{{main-header}}
+{{#es-header}}
+  {{#es-navbar links=links}}
```

Comment:
Can we have `es-navbar` contain all the links internally within itself?

Reply:
yes we did discuss this a few times and I can't remember which side we landed on 🤔 @MelSumner @rwwagner90 do you have an insight? did we ultimately decide that the navbar would have a default and then you could override it if you wanted to?

Reply:
We've discussed this several times. I am in favor of having the links be default or at least importable from the style guide, but I think @MelSumner would prefer that we not do that, since all the sites will hopefully be one repo eventually and then the other usages would be like ember-cli, fastboot, etc sites, which would have different links, rendering defaults somewhat useless.

Reply:
Because the addon is intended to go into many different apps & repos (not just emberjs.com, but also ember-cli.com, ember-engines.com, fastboot, etc) and have multiple themes, it just doesn't seem like the right long-term move to make.